roottest/root/io/alloc/.rootrc

```diff
@@ -1,2 +0,0 @@
-Rint.History:            .root_hist
-ACLiC.LinkLibs:  1
```

Comment:
This one should probably not be removed.  It changed the value of `ACLiC.LinkLibs` on Windows.

Reply:
I see ... but then all the other removal introduced a change ....